react/src/components/VFolderTable.tsx

```diff
@@ -244,16 +250,17 @@ const VFolderTable: React.FC<VFolderTableProps> = ({
    * @param input - The input path to be converted.
    * @returns The aliased path based on the name and input.
    */
-  const inputToAliasPath = useEventNotStable(
+  const inputToAliasPath = useCallback(
     (name: VFolderKey, input?: string) => {
-      if (_.isEmpty(input)) {
+      if (input === undefined || input === '') {
         return `${aliasBasePath}${name}`;
-      } else if (input?.startsWith('/')) {
+      } else if (input.startsWith('/')) {
         return input;
       } else {
         return `${aliasBasePath}${input}`;
       }
     },
+    [aliasBasePath],
   );
```

Comment:
I understood that the reason you used the useCallback hook is to prevent the component from declaring a function with new memory allocation when the component is re-rendered, but is it always a good idea to declare the function using useCallback even if the computation of the function is not heavy?